### DISCUSSION TITLE
What is the best way to configure fontawesome?

### DISCUSSION THREAD

**Discussion by aaron-za-dev** [primefaces/community#447] [PrimeNG] [answered]
I want to use the fontawesome icons, but in the docs I can't find how to install and configure them

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

1-) Download fontawesome by npm.
2-) Register fontawesome styles into the angular.json.
3-) Go to the component documentation you'd like to implement. Check out the templates section. You'll see different icon templates in every component. We've made all icons customizable by templating. Use the templating as described. 

**For example:**
```
<ng-template pTemplate="closeicon">
    <CloseIcon /> // FontAwesome icon
</ng-template>
```

If you're also asking for `How to configure font awesome in Angular?` please visit the Fontawesome's official documentation.
